# Don't let the make fail when makeinfo is not found.
# The missing script is supposed to handle missing deps for the build,
# but by returning the actual exit code, the build will still fail.
# In GDB 7.8, they always return 0 as exit code. This patch does the 
# same.
# Bug report at https://sourceware.org/bugzilla/show_bug.cgi?id=18113
diff -ur gdb-7.9.orig/missing gdb-7.9/missing
--- gdb-7.9.orig/missing	2015-02-19 12:58:08.000000000 +0100
+++ gdb-7.9/missing	2015-03-11 14:22:52.000000000 +0100
@@ -204,7 +204,7 @@
 
 # Propagate the correct exit status (expected to be 127 for a program
 # not found, 63 for a program that failed due to version mismatch).
-exit $st
+exit 0
 
 # Local variables:
 # eval: (add-hook 'write-file-hooks 'time-stamp)
